osquery/devtools/shell.cpp

```diff
@@ -119,7 +119,12 @@ static char zTimerHelp[] =
 #define MODE_Pretty 5 // Pretty print the SQL results
 
 static const char* modeDescr[] = {
-    "line", "column", "list", "semi", "csv", "pretty",
+    "line",
+    "column",
+    "list",
+    "semi",
+    "csv",
+    "pretty",
```

Comment:
May I ask you to submit formatting changes in separate PR?

Reply:
This sometimes happens when we update the `clang-format ` executable from the brew repository and there are changes on how code is formatted by the tool. If I remove this, then the `make audit` step will fail (and the CI workers will fail too).

These lines will disappear once we rebase on master after the correctly formatted code is committed.

Reply:
That's really cool! Has this been introduced recently? I had several PR fail due to this in the recent past!